_aggregations/metric/scripted-metric.md

```diff
@@ -55,18 +231,38 @@ GET opensearch_dashboards_sample_data_logs/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+
+The response returns three values in the `value` object, demonstrating how a scripted metric can return multiple metrics at once by using a map in the state:
```

Comment:
"scripted metric" => "`scripted_metric` aggregration"? Should "state" be in code font?